chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -124,37 +131,21 @@ object CBPurchase {
             )
         }
     }
-    @JvmStatic
-    @Throws(InvalidRequestException::class, OperationFailedException::class)
-    fun queryPurchaseHistory() {
-        try {
-            billingClientManager?.queryPurchaseHistory()
-        }catch (exp: Exception){
-            Log.i(javaClass.simpleName, "Exception in validateReceipt() :"+exp.message)
-            ChargebeeResult.Error(
-                exp = CBException(
-                    error = ErrorDetail(
-                        exp.message
-                    )
-                )
-            )
-        }
-    }
-    @JvmStatic
-    @Throws(InvalidRequestException::class, OperationFailedException::class)
-    fun queryAllPurchases() {
-        try {
-            billingClientManager?.queryAllPurchases()
-        }catch (exp: Exception){
-            Log.i(javaClass.simpleName, "Exception in validateReceipt() :"+exp.message)
-            ChargebeeResult.Error(
-                exp = CBException(
-                    error = ErrorDetail(
-                        exp.message
-                    )
-                )
-            )
-        }
+
+    fun validateReceipt(purchaseToken: String, productId: String, completion : (ChargebeeResult<Any>) -> Unit){
+        val logger = CBLogger(name = "buy", action = "process_purchase_command")
```

Comment:
Should we change the `action` name?

Reply:
yes @cb-haripriyan , we can do it. any suggestions how the action name should be? currently the `action` name was updated the API name that we use.